config/tikv/aes128-encry.toml

```diff
@@ -0,0 +1,10 @@
+[security.encryption]
```

Comment:
1. we only need aes128 for test purpose. we can remove the aes256 config.
2. rename the file to `encryption.toml`?